.gitmodules

```diff
@@ -10,3 +10,6 @@
 [submodule "homeassistant/components/frontend/www_static/home-assistant-polymer"]
 	path = homeassistant/components/frontend/www_static/home-assistant-polymer
 	url = https://github.com/balloob/home-assistant-polymer.git
+[submodule "homeassistant/external/verisure"]
```

Comment:
I see you already did setup the correct requirements. Just remove the git submodule.